bioio/plugins.py

```diff
@@ -233,6 +233,11 @@ def get_plugins(use_cache: bool) -> Dict[str, List[PluginEntry]]:
     # Save copy of plugins to cache then return
     plugins_by_ext_cache.clear()
     plugins_by_ext_cache.update(plugins_by_ext)
+
+    # Sort dict from longest to shortest key
+    plugins_by_ext = dict(
+        sorted(plugins_by_ext.items(), key=lambda x: len(x[0]), reverse=True)
+    )
```

Comment:
A few concerns about this.
1. you're changing the plugins_by_ext after its previous value was stored in the plugins_by_ext_cache.
2. this is completely undoing the "insert in sorted order by most recently installed" which we did above
3. maybe the least interesting of the three but it might have some implications -- this is a global sort over all file extensions right?  You're not just prioritizing ometiff over tiff, but really scrambling the order.